src/fleet_app/create.cr

```diff
@@ -25,5 +25,14 @@ module FleetApp
         body: body
       )
     end
+
+    def self.create_with_auth(host : String, game_name : String, server_id : String, basic_auth : String, body : String = "", environment : String = "production")
```

Comment:
I think what you can do for this and the following commands is to use `create` command, add `basic_auth` parameter that defaults to "".